### PR TITLE
fix(@desktop/chat): Fix displaying a list of contacts in StatusTagSelector

### PR DESCRIPTION
**Pull request by endulab**
Fix #5465

### Checklist

- [x] follow [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
  - the scope should be the component's name e.g: `feat(StatusListItem): ... `
  - when adding new components, the scope is the module e.g: `feat(StatusQ.Controls): ...`
- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?
- [ ] is this a breaking change?
    - [ ] use the dedicated `BREAKING CHANGE` commit message section
    - [ ] resolve breaking changes in [status-desktop](https://github.com/status-im/status-desktop)
        - [ ] (pre-merge) adapt code to breaking changes
        - [ ] (post-merge) update StatusQ submodule pointer
- [x] test changes in [status-desktop](https://github.com/status-im/status-desktop)

### CODE REVIEW

File: src/StatusQ/Components/StatusTagSelector.qml [hunk 1]
```diff
@@ -139,9 +140,9 @@ Item {
             for (var i = 0; i < inputModel.count; i++ ) {
                 var entry = inputModel.get(i);
                 if (entry.name.toLowerCase().includes(text.toLowerCase())) {
-                    sortedList.insert(sortedList.count, {"publicId": entry.publicId, "name": entry.name,
-                                          "icon": entry.icon, "isIdenticon": entry.isIdenticon,
-                                          "onlineStatus": entry.onlineStatus});
+                    sortedList.append({"publicId": entry.publicId, "name": entry.name,
```
Comment (PascalPrecht) [inline src/StatusQ/Components/StatusTagSelector.qml:143]: Curious why this hasn't been an issue before...

**Review by PascalPrecht — COMMENTED** [review]
Changes LGTM.

@endulab can you update the commit message to have a proper scope? (`StatusTagSelector`)

**Review by alaibe — APPROVED** [review]

**Review by stefandunca — APPROVED** [review]

**Comment by endulab** [comment]
> Changes LGTM.
> 
> @endulab can you update the commit message to have a proper scope? (`StatusTagSelector`)

@PascalPrecht I fixed commit message.

File: src/StatusQ/Components/StatusTagSelector.qml (hunk 1 above)
Comment (endulab) [reply src/StatusQ/Components/StatusTagSelector.qml:143]: This line does not cause any problems. I changed this during researching the problem. append() is a good function to append :)

**Pull request merged**